### ISSUE TITLE
nginx can't load the deployed rails app because of a default configuration left in 'sites-enabled'

### ISSUE THREAD

**Issue by hayesmp** [teohm/rackbox-cookbook#12]
There is a default sym-link ('000-default') in the /etc/nginx/sites-enabled directory that super-cedes the deployed rails app configuration. It needs to be deleted in order for nginx to load the rails app.

**Comment by teohm** [comment]
Thanks @hayesmp I will look into the problem.

**Comment by austinmills** [comment]
Just ran into this myself.

**Comment by hayesmp** [comment]
@austinmills I added a block to delete the file in my fork of the cookbook:

https://github.com/hayesmp/rackbox-cookbook/blob/master/libraries/helpers.rb#L88

**Comment by austinmills** [comment]
Yeah, I pretty much did the same thing, but thought I'd ping the issue anyway to give it a little more weight. :)

**Comment by teohm** [comment]
Hey Austin and Michael, thanks for patching the cookbooks :)

Currently I don't have time to maintain the cookbooks, I wonder if anyone
interested to take over the project? We can chat more about the details to
transfer the ownership.

On Thu, Nov 13, 2014 at 10:41 AM, Austin Mills notifications@github.com
wrote:

> Yeah, I pretty much did the same thing, but thought I'd ping the issue
> anyway to give it a little more weight. :)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/teohm/rackbox-cookbook/issues/12#issuecomment-62832263
> .

## 

Huiming
